File: spore.php
<?php
    $sporeUrl = "https://raw.githubusercontent.com/LafeLabs/spore/refs/heads/main/spore.json";
    $baseUrl = explode("spore.json",$sporeUrl)[0];
    $files = json_decode(file_get_contents($sporeUrl), true);
    foreach ($files as $file) {
        @copy($baseUrl.$file,$file);
    }
?>
<a href = "index.html">index.html</a>
<style>
a{
    font-size:3em;
    color:blue;
    font-family:arial;
}
</style>

File: spore.py
#spore.py
import json
import urllib.request

spore = {}
spore['json'] = 'https://raw.githubusercontent.com/LafeLabs/spore/refs/heads/main/python/spore.json'

files_root = spore['json'].split("spore.json")[0]

with urllib.request.urlopen(spore['json']) as response:
    spore = json.loads(response.read().decode())

for file in spore['files']:
    print(files_root + file)
    urllib.request.urlretrieve(files_root + file, file)